rj_common/include/rj_common/time.hpp

```diff
@@ -48,82 +48,82 @@ constexpr double num_seconds(Duration d) {
 }
 
 /// Converts an integer timestamp in microseconds to a decimal number of seconds
-constexpr float timestamp_to_secs(RJ::Timestamp timestamp) {
+constexpr float timestamp_to_secs(rj::Timestamp timestamp) {
     return (float)(timestamp / 1000000.0f);
 }
 
 }  // namespace RJ
```

Comment:
Probably just semantics but maybe change the comment to namespace rj so it accurately reflects the namespace